topic7/src/main/java/models/NotesDao.java

```diff
@@ -0,0 +1,12 @@
+package models;
+
+import javax.transaction.Transactional;
+
+import org.springframework.data.repository.CrudRepository;
+
+@Transactional
+public interface NotesDao extends CrudRepository<Notes, Long> {
+
+	public Notes findByStudentNameAndLastName(String name, String lastName);

```

Comment:
Please read point 3.3.2. Query creation in http://docs.spring.io/spring-data/jpa/docs/current/reference/html/ to have more information